@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/pfz/fn_pfz.sqf

```diff
@@ -1,3 +1,19 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_pfz
+
+Description:
+    Main pfz function
```

Comment:
What does this mean?

This code actually handles the creation of PFZs. Could rename to `fza_fnc_pfzCreate`.

Good description would be

> Runs the in-map user interface that is used for selecting PFZs. Generates the PFZ for the given helicopter and puts all relevant targets inside it.

You also need to fill out the parameters section, e.g.

```sqf
_heli - the helicopter to create the PFZ on
_pfznum - the index of the PFZ (in range 1 - 8 inclusive)